src/openforms/variables/tests/test_validators.py

```diff
@@ -184,3 +185,80 @@ def test_it_invalidates_field_names_invalid_characters(
 
         with self.assertRaisesMessage(ValidationError, unquoted_repr(faulty_name)):
             self.validate(faulty_input)
+
+
+class QueryParameterValidatorTests(SimpleTestCase):
+    validate = QueryParameterValidator()
+
+    def test_validate_none_success(self):
+        value = None
+
+        self.validate(value)
+
+    def test_validate_success(self):
+        value = {
+            "foo": ["bar"],
+            "bar": ["baz", "qux"],
```

Comment:
What does requests do when you pass a dict of lists of multiple strings? I couldn't find it in the docs, and I only remember passing it `list[tuple[str, str]]` or `dict[str, str]`.

The RFC standard doesn't prescribe any semantics. I know php parses a querysting "foo[]=1,foo[]=2" as an array foo. But I've seen Java services that do something with semi colons.

Reply:
Tested this real quick, it seems that doing `requests.get(url, params={"foo": ["bar", "baz"]})` results in a querystring like this: `?foo=bar&foo=baz`